<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRegister</class>
 <widget class="QDialog" name="DialogRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #f5f5f5;
        font-family: &quot;Segoe UI&quot;;
        font-size: 14px;
    }
    QGroupBox {
        border: 2px solid #0078d7;
        border-radius: 8px;
        margin-top: 10px;
        padding-top: 15px;
        font-size: 16px;
        font-weight: bold;
        color: #0078d7;
    }
    QLabel {
        color: #333333;
    }
    QLineEdit, QComboBox, QDateEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 5px;
        font-size: 14px;
    }
    QLineEdit:focus, QComboBox:focus, QDateEdit:focus {
        border: 1px solid #0078d7;
    }
    QPushButton {
        background-color: #0078d7;
        color: white;
        border: none;
        border-radius: 4px;
        padding: 8px 16px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #005bb5;
    }
    QPushButton:pressed {
        background-color: #004080;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>注册信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelEmployeeID">
        <property name="text">
         <string>工号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditEmployeeID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelGender">
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxGender"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>日期：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDepartment">
        <property name="text">
         <string>部门：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxDepartment"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelPosition">
        <property name="text">
         <string>职位：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxPosition"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelTelephone">
        <property name="text">
         <string>电话：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditTelephone"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>邮箱：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelPhoto">
        <property name="text">
         <string>照片：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditPhoto"/>
        </item>
        <item>
         <widget class="QPushButton" name="BtnBrowse">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelRole">
        <property name="text">
         <string>用户角色：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="comboBoxRole"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="BtnSubmit">
       <property name="text">
        <string>提交</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogRegister</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
